vignettes/function-flow.Rmd

```diff
@@ -95,63 +95,202 @@ outputs.](images/function-flow-population.png)
+    -   `is_t2d`: Define whether the diagnosis was T2D-specific
+        (`c_diag` starts with "DE11" or "250")
+    -   `department`: Define whether the diagnosis was made made by an
+        endocrinological (`c_spec` == 8) or other medical department
+        (`c_spec` \< 8 or 9-30)
+-   LPR3:
+    -   `pnr`: identifier variable
+    -   `do_diagnosis`: include all diabetes diagnoses, registered as
+        primary (A) or secondary (B) diagnoses, regardless of type or
+        department: `diagnosekode` starts with "DE1[0-4]" and
+        `diagnosetype` is either "A" or "B", but exclude retracted
+        diagnoses (`senere_afkraeftet` == "Ja")
+    -   `is_primary`: Define whether the diagnosis was a primary
+        diagnosis (`diagnosetype` == "A")
+    -   `is_t1d`: Define whether the diagnosis was T1D-specific
+        (`diagnosekode` starts with "DE10")
+    -   `is_t2d`: Define whether the diagnosis was T2D-specific
+        (`diagnosekode` starts with "DE11")
+    -   `department`: Define whether the diagnosis was made made by an
+        endocrinological (`hovedspeciale_ans` == "medicinsk
+        endokrinologi") or other medical department (`hovedspeciale_ans`
+        either "Blandet medicin og kirurgi", "Intern medicin",
+        "Geriatri", "Hepatologi", "Hæmatologi", "Infektionsmedicin",
+        "Kardiologi", "Medicinsk allergologi", "Medicinsk
+        gastroenterologi", "Medicinsk lungesygdomme", "Nefrologi",
+        "Reumatologi", "Palliativ medicin", "Akut medicin",
+        "Dermato-venerologi", "Neurologi", "Onkologi", "Fysiurgi", or
+        "Tropemedicin")
+
+These intermediate results are combined for further processing, and
+`include_diabetes_diagnoses()` outputs a single `data.frame` with the
+following variables (one row for each individual):
+
+-   identifier variable (`pnr`)
+-   date of the first diabetes diagnosis (`do_diagnosis_1`)
+-   date of the second diabetes diagnosis (`do_diagnosis_2`)
+-   number of type 1 diabetes-specific primary diagnosis codes from
+    endocrinological departments (`n_t1d_endo`)
+-   number of type 2 diabetes-specific primary diagnosis codes from
+    endocrinological departments (`n_t2d_endo`)
+-   number of type 1 diabetes-specific primary diagnosis codes from
+    medical departments (`n_t1d_medical`)
+-   number of type 2 diabetes-specific primary diagnosis codes from
+    medical departments (`n_t2d_medical`)
+
+The output is passed to the `get_diagnosis_date()` function for the
+final step of the inclusion process and is subsequently used to classify
+diabetes type.
 
 #### Diabetes-specific podiatrist services
 
 The function `include_podiatrist_services()` uses `sysi` or `sssy` as
 input to extract the dates of all diabetes-specific podiatrist services.
 
-<!-- TODO: Add details on how this filtering should be done -->
+These dates are extracted by filtering values beginning with "54" in the
+`spec` variable of the `sssy` and `sysi` registers by default
```

Comment:
In `variable_description`, this variable is called `speciale´. Is this discrepancy a mistake in the description or here?

Reply:
You're right - my mistake :)